mappings/net/minecraft/potion/Potions.mapping

```diff
@@ -21,3 +21,9 @@ CLASS net/minecraft/unmapped/C_bvmvcjvn net/minecraft/potion/Potions
 	FIELD f_wdebhrfm LONG_STRENGTH Lnet/minecraft/unmapped/C_cjzoxshv;
 	FIELD f_wdkfcvgu LONG_SLOW_FALLING Lnet/minecraft/unmapped/C_cjzoxshv;
 	FIELD f_whpdzgwd HARMING Lnet/minecraft/unmapped/C_cjzoxshv;
+	METHOD m_oaevlkxc registerByKey (Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_almkolgw;)Lnet/minecraft/unmapped/C_cjzoxshv;
```

Comment:
```suggestion
	METHOD m_oaevlkxc registerByKey (Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_almkolgw;)Lnet/minecraft/unmapped/C_cjzoxshv;
```
since this is functionally identical to the other `register` method, I would let the parameters speak for themselves